src/PKSim.Infrastructure/Services/PopulationExportTask.cs

```diff
@@ -238,12 +238,14 @@ private bool settingsRequired(Simulation simulation)
          return !simulation.OutputSelections.HasSelection;
       }
 
-      private void addParameterToTable(IVectorialParametersContainer parameterContainer, DataTable dataTable, IParameter parameter)
+      private void addParameterToTable(IVectorialParametersContainer parameterContainer, DataTable dataTable, IParameter parameter, bool includeUnitsInHeader)
       {
          //some path have changed and the parameter is not found anymore
          if (parameter == null) return;
 
-         addColumnForParameterToTable(parameterContainer, dataTable, _entityPathResolver.PathFor(parameter), parameter.Dimension.BaseUnit.Name);
+         var unitName = includeUnitsInHeader ? parameter.Dimension.BaseUnit.Name : string.Empty;
```

Comment:
Suggestion is to create a columnName here instead of passing an empty unitName if not requried. 
Then rename the parameter in `addColumnForParameterToTable` to columnName